src/ui/components/SearchInput/style.scss

```diff
@@ -46,6 +43,15 @@ $transition-duration: 250ms;
     opacity: 1;
     // Use !important to overwrite style properties added in JS.
     transform: translateX(0) !important;
+  }
+}
+
+.SearchInput--animating {
+  .SearchInput-animation-icon {
+    transition: opacity 1ms $transition-duration, transform $transition-duration;
```

Comment:
Oh gosh, right! I totally knew this but forgot. I think the comment helps because not everyone is familiar with that delay value (and those like me who know but forget will have their memory jogged). 👍